JavaScript/js/messengerDefinition.js

```diff
@@ -1,24 +1,48 @@
-// Jacob's working on this...
+// JD's working on this...
```

Comment:
If we want to refer to the code as just "The code" we might remove these types of lines from future versions.